### PR TITLE
put learns tag for all page

### PR DESCRIPTION
**Pull request by tlai**
Duplicated all learns tag into all page of GPS

### CODE REVIEW

File: api/html5_geolocation.js [hunk 1]
```diff
@@ -58,220 +59,229 @@ Geolocation = {
...
-	 * @description Error code. One of the PERMISSION_DENIED, POSITION_UNAVAILABLE or TIMEOUT 
-	 * @BB50+
-	 * @PB10	
-	 * @readonly	 
-	 * @type Short
-	 */
+PositionError  = {
+
+    /**
+     * @description Error code. One of the PERMISSION_DENIED, POSITION_UNAVAILABLE or TIMEOUT
```
Comment (jeffheifetz) [inline api/html5_geolocation.js]: Could you make these link tags?

**Comment by jeffheifetz** [comment]
How come it looks like a huge diff when most lines look the same?

**Comment by tlai** [comment]
i also removed trailing space and convert TAB to Space.

**Comment by tlai** [comment]
Good idea. Done

**Comment by jeffheifetz** [comment]
I guess we have to switch to spaces eventually, so might as well start, its just annoying we lost all tracking. Looks good to me

**Pull request merged**